api/countdown.py

```diff
@@ -1,39 +1,22 @@
-from telegram import Update, Bot
 from datetime import datetime
 import dateparser
 
 
-def countdown(bot: Bot, update: Update):
+def countdown(update, context):
+    """Command to show days left till a date."""
     message = update.message
-    try:
-        to = message.text.strip().split(' ', 1)[1]
-    except IndexError:
-        bot.send_message(
-            chat_id=message.chat_id,
-            text="*Usage:* `/hltb {GAME_NAME}`\n"
-                 "*Example:* `/hltb horizon zero dawn`",
-            reply_to_message_id=message.message_id,
-            parse_mode='Markdown'
-        )
-        return
+    countdown_to = ' '.join(context.args)
 
-    future = dateparser.parse(to)
-    today = datetime.now()
+    if not countdown_to:
+        text = "*Usage:* `/countdown {WHENEVER}`\n"\
+               "*Example:* `/countdown 9 november`"
+    else:
+        future = dateparser.parse(countdown_to)
 
-    delta = future - today
+        if future:
+            delta = future - datetime.now()
+            text = f"{abs(delta.days)} days {'a' if delta.days < 0 else 'to '}go",
+        else:
+            text = "Invalid date format"
 
-    if delta.days > 0:
-        otuput = f'{delta.days} days to go'
-        bot.send_message(
-            chat_id=message.chat_id,
-            text=otuput,
-            reply_to_message_id=message.message_id,
-            parse_mode='Markdown',
-        )
-    else:
-        bot.send_message(
-            chat_id=message.chat_id,
-            text=f'{abs(delta.days)} days ago',
-            reply_to_message_id=message.message_id,
-        )
-        return
+    context.bot.send_message(chat_id=message.chat_id, text=text)
```

Comment:
Currently replies: `"38 days ago"`. Remove the inverted commas to get: `38 days ago`